### PR TITLE
G-API: Suppress ngraph warnings for MSVS2015 (4268)

### PR DESCRIPTION
**Pull request by mpashchenkov**
Relates #21134

OpenVINO recommends to use MSVS 2019.  Warnings (4268) are disabled.

```
force_builders=Custom Win
build_image:Custom Win=openvino-2021.4.2-vc14
test_modules:Custom Win=dnn,gapi,python2,python3,java
```

### CODE REVIEW

File: modules/gapi/test/infer/gapi_infer_ie_test.cpp [hunk 1]
```diff
@@ -29,6 +29,9 @@
 #elif defined(_MSC_VER)
 #pragma warning(push)
 #pragma warning(disable : 4100)
+# if _MSC_VER < 1900
+#  pragma warning(disable:4268) // Disable warnings of ngraph. OpenVINO recommends use MSVS 2019.
```
Comment (alalek) [inline modules/gapi/test/infer/gapi_infer_ie_test.cpp:33]: `1900` is MSVS 2015

This condition doesn't work at all.

Comment (mpashchenkov) [reply modules/gapi/test/infer/gapi_infer_ie_test.cpp:33]: "1900 is MSVS 2015" - I know. Local build was successful with this condition. But OV package was fresher.
Sorry for experiment.

**Comment by alalek** [comment]
@mpashchenkov What is the status of this PR? Is it ready to be merged?

**Comment by mpashchenkov** [comment]
@alalek, I think yes. PR contains simple disabling for two msvs2015 warnings. Origin - ngraph's header. Does PR need one more review from someone?

**Review by alalek — APPROVED** [review]

**Pull request merged**